native_chrome_extension/content.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Function to extract relevant information from the document
+*/
+function extractDocumentInfo() {
+  const documentInfo = {
+    title: document.title,
+    url: window.location.href,
+    html: document.documentElement.outerHTML
+  };
+  return documentInfo;
+}
+
+
+/*
+ * Send the document info to the background script
+*/
+function sendDocumentInfo() {
+    const documentInfo = extractDocumentInfo();
+    chrome.runtime.sendMessage({ type: 'document', documentInfo });
+}
+
+
+sendDocumentInfo();
+document.addEventListener('DOMSubtreeModified', sendDocumentInfo);
```

Comment:
Alternative to Mutation Observer but use DOM Subtree Modified.

Reply:
Can you please list tradeoffs between these approaches?